src/main/scala/org/deepdive/helpers/Helpers.scala

```diff
@@ -171,4 +172,23 @@ object Helpers extends Logging {
         s""" --silent -N -e '${query.replaceAll("'", "'\\\\''")}' """
     }
   }
+
+
+  def slugify(input: String): String = {
```

Comment:
Unit tests for these as well.  Not sure why this helper is needed from where.  Would be helpful if there's a brief comment.